### DISCUSSION TITLE
Plot Delay Percentage

### DISCUSSION THREAD

**Discussion by Abro86** [Chia-Network/chia-blockchain#2290] [Ideas]
As the title states I'm just suggesting the option to set the plot delay as a percentage instead of time in minutes. It seems like this would take some guess work out. Thoughts?

**Comment by cccat6** [comment]
You need to do experiments. Then use the data of CPU usage, RAM usage, SSD usage, etc to make some charts and do some math work. Then you can find the optimal setting of everything. Wach out your bottleneck. It usually the thing that limits you plotting more at the same time. If the throughput of your SSD is the bottleneck. It will be disappointed. Good luck.

**Reply by Abro86** [reply]
I'm just noticing in version 1.0.5 at least, that at 31% plotting exits phase 1 and begins phase 2. I've replicated this on alternative hardware and others have reported the same. Being able to set the stagger to 31% in this case would remove the need to find the time in minutes that each plotting station would need to be set at.

---------

I'd like to add that the time for phase 1 has been different for each of these workstations, but it has always been at 31% that the phase changes.

**Reply by cccat6** [reply]
My personal experience. Do not use the GUI and Windows. Using Linux and CLI are much easier and more efficient.
For each task, It will take a lot of system resources including everything. It will go down significantly after a couple of hours. That should be after phase 2. You can try to optimize the gap between each plotting task to maximize utilize the system resources.
If you write your own plotting control script, it will be easy.
For example, I have a plotting matching with 3900X and 2*2TB SSDs. I can run 8 tasks on each SSD if my gap is 30min. It was limited by the capacity of SSD. However, if I set the gap been 45min. I will be able to run 9 tasks on a single SSD at the same time, because previous tasks after phase 2 will release some space so I can put one more task parallel.
You should not expect the plotting time increases with the gap is a straight line. It actually a kind of exponential. However, even the average plotting time increases in parallel tasks, the TB/unit-of-time also increases.
All in all, do some experiments. Not just calculate with ideal situation or with few data. You need to spend 2 or 3 days to investigate your best settings.
